### PR TITLE
Create readme

### PR DESCRIPTION
**Pull request by TankevychO**

### CODE REVIEW

**Review by begemot1987 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,42 @@
...
+#  🚖 **Taxi service**
+The application describes the operation of the taxi service. </br>
+To implement the application were created: </br>
+- taxi_service database
+- models: Car, Driver, Manufacturer
+- interfaces DAO and implements for each model
+- interfaces "Services" and implements for each model and authentication
+- logger in the AuthenticationService
+- filter for authentication driver
+- controllers for all actions
```
Comment (Nick97-git) [inline README.md]: let's move this info to the paragraph `Implementation details and technologies` and add this info not as separate paragraphs but as text info, and make it make it shorter

File: README.md [hunk 2]
```diff
@@ -1 +1,42 @@
...
+#### Performs the following actions:
```
Comment (Nick97-git) [inline README.md]:
```suggestion
#### Available actions of the application
```

File: README.md [hunk 3]
```diff
@@ -1 +1,42 @@
...
+![This is an image](https://progi.pro/media/main/f5/f0/11/f5f01101de396d5c76a8eb66efaf9653.png)
+_______________________________________________________________________________________________________________________
+#### Setup "Taxi service"
+- [x] Install MySQL and MySQL Workbench
+- [x] Configure Apache Tomcat
+- [x] In `pom.xml` change to the absolute path         
+  `PATH TO checkstyle.xml` </br>
+  ![This is an image](http://joxi.ru/DmB6xRGcqwgkxm.jpg)
+- [x] In `log4j2.xml` change to the absolute path
+  `PATH TO app.log`
```
Comment (Nick97-git) [inline README.md]:
```suggestion
- [x] Change absolute path to your log file in `log4j2.xml` 
```

File: README.md [hunk 4]
```diff
@@ -1 +1,42 @@
...
+_______________________________________________________________________________________________________________________
+#### Implementation details and technologies
+Project based on 3-layer architecture:</br>
+![This is an image](https://progi.pro/media/main/f5/f0/11/f5f01101de396d5c76a8eb66efaf9653.png)
+_______________________________________________________________________________________________________________________
+#### Setup "Taxi service"
+- [x] Install MySQL and MySQL Workbench
+- [x] Configure Apache Tomcat
+- [x] In `pom.xml` change to the absolute path         
+  `PATH TO checkstyle.xml` </br>
```
Comment (Nick97-git) [inline README.md]:
```suggestion
- [x] Change the absolute path to checkstyle.xml in `pom.xml` </br>
```

File: README.md [hunk 5]
```diff
@@ -1 +1,42 @@
...
+  ![This is an image](http://joxi.ru/DmB6xRGcqwgkxm.jpg)
+- [x] In `log4j2.xml` change to the absolute path
+  `PATH TO app.log`
+  ![This is an image](http://joxi.ru/zANxkBLU1B8okm.jpg)
+- [x] Run script from the resources/init_db.sql file in the Workbench.
+- [x] In `/util/ConnectionUtil.java` change
+  `URL`, `USERNAME`, `PASSWORD`, `JDBC_DRIVER`
```
Comment (Nick97-git) [inline README.md]:
```suggestion
- [x] Change `URL`, `USERNAME`, `PASSWORD`, `JDBC_DRIVER` in `/util/ConnectionUtil.java` 
```

**Review by PaulMartsiniv — APPROVED** [review]

**Review by Nick97-git — CHANGES_REQUESTED** [review]
- DeleteCarController, DeleteDriverController, DeleteManufacturerController -> do you have /cars/all, /drivers/all, /manufacturers/all url patterns?
- remove redundant `/` in the end of url patterns like /cars/, /drivers/, /manufacturers/
- DriverDaoImpl -> move your private method after public ones
- AuthenticationServiceImpl -> remove `else`, it's redundant

**Comment by Nick97-git** [comment]
One more comment. At the moment your path looks like src/main/java/mate... We should rename "mate" to something else, like "taxi". After you do that don't forget to change the value you pass to the Injector.getInstance() everywhere you use it

File: README.md [hunk 6]
```diff
@@ -1 +1,32 @@
...
+Project based on 3-layer architecture:</br>
+![This is an image](https://progi.pro/media/main/f5/f0/11/f5f01101de396d5c76a8eb66efaf9653.png)
+To implement the application were created: database, models, interfaces DAO and services for each model, logger, filter, controllers for all actions.
+_______________________________________________________________________________________________________________________
+#### Setup "Taxi service"
+- [x] Install MySQL and MySQL Workbench
+- [x] Configure Apache Tomcat
+- [x] Change the absolute path to checkstyle.xml in `pom.xml` </br>
+  ![This is an image](http://joxi.ru/DmB6xRGcqwgkxm.jpg)
+- [x] Change absolute path to your log file in `log4j2.xml` </br>
```
Comment (Nick97-git) [inline README.md]:
```suggestion
- [x] Change the absolute path to your log file in `log4j2.xml` </br>
```

File: src/main/resources/log4j2.xml [hunk 7]
```diff
@@ -8,7 +8,7 @@
         </File>
     </Appenders>
     <Loggers>
-        <Logger name="mate.academy" level="info" additivity="false">
+        <Logger name="taxi.academy" level="info" additivity="false">
```
Comment (Nick97-git) [inline src/main/resources/log4j2.xml]: remove academy)

**Review by Nick97-git — CHANGES_REQUESTED** [review]

**Review by Nick97-git — APPROVED** [review]

**Pull request merged**